### PR TITLE
Add KVM and tunnel devices to the container when launching a VM

### PR DESCRIPTION
**Pull request by gabrielhartmann**
In order to launch virtual machines in a Titus container the devices `/dev/kvm` and `/dev/net/tun` must be added.

### CODE REVIEW

File: executor/runtime/docker/capabilities.go [hunk 1]
```diff
@@ -30,36 +30,49 @@ func addAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.Hos
 }
 
 func setupAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.HostConfig) error {
-	addedCapabilities := addAdditionalCapabilities(c, hostCfg)
+	if c.TitusInfo.GetAllowNestedContainers() {
+		return errors.New("nested containers no longer supported")
```
Comment (sargun) [inline executor/runtime/docker/capabilities.go]: If you want this can be defined as a var

File: executor/runtime/docker/capabilities.go [hunk 2]
```diff
@@ -30,36 +30,49 @@ func addAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.Hos
...
+	}
 
-	// Privileged containers automaticaly deactivate seccomp and friends, no need to do this
 	fuseEnabled, err := c.GetFuseEnabled()
+	if err != nil {
+		return err
+	}
 
+	kvmEnabled, err := c.GetKvmEnabled()
```
Comment (sargun) [inline executor/runtime/docker/capabilities.go]: Do KVM containers need CAP_SYS_ADMIN?

Comment (gabrielhartmann) [reply executor/runtime/docker/capabilities.go]: no, not to launch the VM anyway post network configuration

Comment (sargun) [reply executor/runtime/docker/capabilities.go]: I think for the network config, all you need is CAP_NET_ADMIN.

Comment (gabrielhartmann) [reply executor/runtime/docker/capabilities.go]: Yes

**Pull request merged**